src/backtrace.h

```diff
@@ -191,6 +191,14 @@ void collect_backtrace(void) {
         backtrace_gdb();
 #else
         fprintf(stderr, "Backtrace support through gdb is not available.\n");
+#endif
+    } else if (0 == strcmp(method, "auto")) {
```

Comment:
I wonder here if we should support "yes" or any other string(s) here.  If a user sets BACKTRACE_METHOD=1, or anything that's not "execinfo" or "gdb", should we just fall into the "auto" case?  This is just a thought, it might be undesirable/overkill.

Reply:
@davidozog If the user sets anything other than "execinfo"/"gdb"/"auto", the "else" block will be executed and it will check for execinfo. If execinfo is not available, then it is exiting with a message. This behavior is slightly different than falling into "auto" (gdb will not be checked). Let me know if you guys think "auto" should be better option for invalid user input.

Reply:
Sounds good, I'm ok with either way too.

It would be good to clarify in the README that "auto" is the valid input for (i.e. "Options are : auto, execinfo, gdb").  I think some vars use the "(default: auto)" description where "auto" is not actually the input string (like `SHMEM_OFI_PROVIDER`).

Reply:
Right. Thanks for noting that. I was also thinking whether we should have something on backtracing in the Wiki. As we add more methods in the future, I think it will be better if we have a small description on the Wiki page.